packages/testing/src/test/chai-retry-plugin.unit.ts

```diff
@@ -0,0 +1,295 @@
+import Chai, { expect } from 'chai';
+import { chaiRetryPlugin } from '../chai-retry-plugin/chai-retry-plugin';
+
+Chai.use(chaiRetryPlugin);
+
+describe('chai-retry-plugin', () => {
+    it('should retry a function that eventually succeeds', async () => {
+        let attempts = 0;
+
+        const funcToRetry = () => {
+            attempts++;
+            if (attempts < 3) {
+                throw new Error('Failed');
+            }
+            return 'Success';
+        };
+
+        await expect(funcToRetry).to.retry({ retries: 5, delay: 10 });
```

Comment:
for the base case happy path not set any retry options

Reply:
@yurii-ve fixed